### PR TITLE
Update elfinder.uk.js

### PR DESCRIPTION
**Pull request by da411d**

### CODE REVIEW

File: js/i18n/elfinder.uk.js [hunk 1]
```diff
@@ -245,8 +245,8 @@ if (elFinder && elFinder.prototype && typeof(elFinder.prototype.i18) == 'object'
 			'moveFiles'       : 'Перемістити файли',
 			'copyFiles'       : 'Копіювати файли',
 			'rmFromPlaces'    : 'Вилучити з розташувань',
-			'untitled folder' : 'неназвана папка',
-			'untitled file.txt' : 'неназваний файл.txt',
+			'untitled folder' : 'Нова папка',
+			'untitled file.txt' : 'new.php',
```
Comment (nao-pon) [inline js/i18n/elfinder.uk.js]: extension must be '.txt'. Did you mean "Нова файл.txt"?

Comment (da411d) [inline js/i18n/elfinder.uk.js]: "Новий файл".
But in my opinion it is more useful to make "new.php". Or somple ".txt", without name)

Comment (nao-pon) [inline js/i18n/elfinder.uk.js]: Extension of the text file is generally a ".txt". If you wish the ".php", please be overwritten in the following manner.

```html
<script src="js/i18n/elfinder.uk.js"></script>
<script>
(function(){
	var mes_uk;
	if (typeof elFinder.prototype.i18.uk != "undefined") {
		mes_uk = elFinder.prototype.i18.uk.messages;
		mes_uk['untitled file.txt'] = 'new.php'; // over write
		// and more...
	}
})();
</script>
```

**Comment by nao-pon** [comment]
Please use [translation tool](http://studio-42.github.io/elFinder/tools/langman/#2.1:uk) for a complete translation. It can generate code when you click the "make it" button after editing.

**Pull request closed without merging**